qml/PingSlider.qml

```diff
@@ -10,14 +10,55 @@ RowLayout {
     property var text
     property double value: 0
     property double from: 0
-    property double to: 0
-    property double ticksNumber: 6
+    property double to: model ? model.length - 1 : 0
+    property double ticksNumber: model ? to : 6
     property alias control: sliderControl
-    property var valueText: undefined
+    property var model: undefined
+    property var modelValue: undefined
+    property alias modelIndex: root.value
 
-    onValueChanged: {
-        spinBox.value = value
-        sliderControl.value = value
+    onModelValueChanged: {
```

Comment:
A little documentation on each step will be nice :)

Reply:
Done.